Update README.adoc (#269)
OpenLiberty/guide-jpa-intro: README.adoc

This configuration adds three required Derby dependencies to the [hotspot=dependencies file=0]`dependencies` configuration so Maven can download the Derby libraries locally. The [hotspot=copyDependencies file=0]`copyDependencies` configuration instructs the Liberty Maven plug-in to copy the Derby libraries to the Liberty shared resources directory that is specified through the [hotspot=location file=0]`location` configuration, and is referenced in the [hotspot=shared-dir file=1]`derbyJDBCLib` `library` configuration of the `server.xml` file.

> In your dev mode console for the `backendServices` microservice, type `r` and press `enter/return` key to restart the Liberty instance for picking up the Derby libraries.

Reviewer: ```suggestion
In the terminal where you started the `backendServices` microservice, type `r` and press the `enter/return` key to restart the Liberty instance and pick up the Derby libraries.
```
